test/data/mock-state.json

```diff
@@ -683,11 +683,73 @@
         "decimals": "18"
       }
     ],
-    "contractExchangeRates": {
```

Comment:
👋  The contractExchangeRates has been removed and replaced by marketData in a lot of places, does this need a migration?

Reply:
i believe a Redux state migration won't be necessary for this case, given that the marketData property encompasses not only contractExchangeRates but also a variety of other elements. Additionally, the structure of marketData is distinct due to its aggregation by chainId. Typically, migrations are beneficial when modifications to the existing data are needed to accommodate a new structure. However, in this instance, there's a significant amount of data absent that would require fetching via an API call to complete the transition.